cloud/pkg/controllermanager/edgeapplication/overridemanager/envoverrider.go

```diff
@@ -0,0 +1,218 @@
+/*
+CHANGELOG
+KubeEdge Authors:
+- refactor envOverrider as a struct that implements the Overrider interface
```

Comment:
Need to indicate where it was refactored from, karmada community?

Reply:
karmarda does not implement a differentiated configuration of env and resource. How do I write copyright here?

Reply:
Okey, I think just adding the kubeedge apache license header is enough

Reply:
```go
/*
Copyright 2021 The KubeEdge Authors.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/
```
Is the above content added

Reply:
Yes :)